controller/commentsController.js

```diff
@@ -123,8 +121,6 @@ exports.deleteComment = async (req, res, next) => {
       );
     }
   } catch (error) {
-    return next(
-      new CustomError(500, "Something went wrong,please try again", error)
-    );
+    return next(error);
```

Comment:
Did you mean to remove the CustomError class?

Reply:
Nahh... it was part of the test I was checking to see how we can improve coverage